### PR TITLE
JAMES-2606 Stop looking for 'objectstorage' file, look for 'objectstore' instead

### PR DESCRIPTION
**Pull request by blackheaven**

### CODE REVIEW

File: server/container/guice/blob-objectstorage-guice/src/main/java/org/apache/james/modules/objectstorage/ObjectStorageBlobConfiguration.java [hunk 1]
```diff
@@ -36,7 +36,7 @@
 
 public class ObjectStorageBlobConfiguration {
 
-    private static final String OBJECTSTORAGE_CONFIGURATION_NAME = "objectstorage";
+    private static final String OBJECTSTORAGE_CONFIGURATION_NAME = "objectstore";
```
Comment (chibenwa) [inline server/container/guice/blob-objectstorage-guice/src/main/java/org/apache/james/modules/objectstorage/ObjectStorageBlobConfiguration.java]: isn't it  "blobstore.properties"?

**Comment by trantienduchn** [comment]
test this please, rabbitmq concurrent test

**Comment by hoangdat** [comment]
Build triggered. sha1 is merged.

**Comment by hoangdat** [comment]
Build started sha1 is merged.

**Comment by hoangdat** [comment]
Build finished. No test results found.

**Comment by hoangdat** [comment]
Build triggered. sha1 is merged.

**Comment by hoangdat** [comment]
Build started sha1 is merged.

**Comment by rouazana** [comment]
test this please

**Comment by rouazana** [comment]
test this please

**Pull request merged**